content/en/tracing/setup/go.md

```diff
@@ -139,6 +139,31 @@ func main() {
 
 For more tracer settings, see available options in the [configuration documentation][57].
 
+### B3 Headers Extraction and Injection
+
+Datadog APM tracer supports [B3 headers extraction][58] and injection for distributed tracing.
+
+Distributed headers injection and extraction is controlled by
+configuring injection/extraction styles. Currently two styles are
+supported:
```

Comment:
Should we just: 
```suggestion
supported: `Datadog` and `B3`.
```
And drop the list below? I'm not sure if it helps much.

Reply:
Yeah, I think that's fine.
A list makes sense if we ever add more, but we don't need to format the docs in anticipation of that.